Resources/Prototypes/Alerts/alerts.yml

```
@@ -407,19 +407,43 @@
   onClick: !type:RetakeVow  { }
   description: alerts-vow-broken-desc
 
+# Goobstation start
+# Changed pulling to accomodate grabbing severity
 - type: alert
   id: Pulled
-  icons: [ /Textures/Interface/Alerts/Pull/pulled.png ]
-  onClick: !type:StopBeingPulled  { }
+  icons:
+  - sprite: /Textures/_White/Interface/Alerts/pull.rsi
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)